engine.py

```diff
@@ -120,9 +128,10 @@ def post_app_init(self):
 
                 # populate a callback map. this is a map of command ids to a
                 # corresponding callback. these are used by the menu and shelf
-                # for executing installed app commands. 
-                self._callback_map = \
-                    dict((cmd.get_id(), cmd.callback) for cmd in commands)
+                # for executing installed app commands.
+                self._callback_map = dict(
+                    (cmd.get_id(), cmd.callback) for cmd in commands
+                )
```

Comment:
Change to use dictionary comprehension syntax too since formatter picked this up

```suggestion
                self._callback_map = {
                    cmd.get_id(): cmd.callback for cmd in commands
                }
```